### DISCUSSION TITLE
Sender IP Addresses

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3245] [Development discussions]
Hello,

I send emails via API. But how can I choose from which IP address the email will be sent when sending?

**Comment by willpower232** [comment]
If you set up an IP Pool then that can force Postal to use specific ones for specific organisations, but you can't specify it when sending an email unless you use IP Pools and have a lot of duplication so you use a specific API key for each IP address which sounds quite messy.
